source/features/repo-age.tsx

```diff
@@ -42,16 +49,17 @@ const getRepoAge = async (commitSha: string, commitsCount: number): Promise<[str
 	return [committedDate, resourcePath];
 };
 
-const getFirstCommit = cache.function(async (): Promise<[string, string]> => {
+// eslint-disable-next-line import/prefer-default-export
+export const getLatestCommitInfo = cache.function(async (): Promise<LatestCommitInfo> => {
```

Comment:
Should this be in its own file?

Reply:
Nah, queries like this grow and disappear as needed, they’re not general-purpose enough. Extracting them would make it seem like it’s the official way of getting the commit info, which it isn’t.